From 615409e15f67ab559066ee2b2b3d2132dc53b0a9 Mon Sep 17 00:00:00 2001
From: Alex Dorchak <halfdome@fluke.com>
Date: Mon, 24 Oct 2016 08:54:47 -0700
Subject: [PATCH 5/5] Driver modification spidev allow generic spidev

---
 drivers/spi/spidev.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/spi/spidev.c b/drivers/spi/spidev.c
index 8fab566..a9db602 100644
--- a/drivers/spi/spidev.c
+++ b/drivers/spi/spidev.c
@@ -707,6 +707,7 @@ static struct class *spidev_class;
 #ifdef CONFIG_OF
 static const struct of_device_id spidev_dt_ids[] = {
 	{ .compatible = "rohm,dh2228fv" },
+	{ .compatible = "spidev" },
 	{},
 };
 MODULE_DEVICE_TABLE(of, spidev_dt_ids);
-- 
1.8.4.5

