From 76a7767556db115371e06a4f5fbe9d7e616cc685 Mon Sep 17 00:00:00 2001
From: Alex Dorchak <halfdome@fluke.com>
Date: Wed, 21 Sep 2016 13:26:44 -0700
Subject: [PATCH 30/30] Fixed a bug in the fluke keypad

---
 drivers/input/keyboard/fluke_keypad.c | 24 ++++--------------------
 1 file changed, 4 insertions(+), 20 deletions(-)

diff --git a/drivers/input/keyboard/fluke_keypad.c b/drivers/input/keyboard/fluke_keypad.c
index 94c5ee4..f987166 100644
--- a/drivers/input/keyboard/fluke_keypad.c
+++ b/drivers/input/keyboard/fluke_keypad.c
@@ -291,24 +291,8 @@ fail:
 	return ret;
 }
 
-static int fluke_keypad_remove(struct device *dev)
-{
-	struct fluke_keypad_data *drv_data = dev_get_drvdata(dev);
-	dev_t dev_num;
-
-	if(drv_data) {
-		cdev_del(&drv_data->cdev);
-		dev_num = MKDEV(fluke_keypad_major, drv_data->minor);
-		unregister_chrdev_region(dev_num, 1);
-		free_irq(drv_data->irq, dev);
-		kfree(drv_data);
-	}
-	
-	return 0;
-}
-
-static const struct of_device_id fluke_gpib_of_match[] = {
-    { .compatible = "flk,fgpib-4.0"},
+static const struct of_device_id fluke_keypad_of_match[] = {
+    { .compatible = "flk,fkeypad-1.0"},
     { {0} }
 };
 MODULE_DEVICE_TABLE(of, fluke_gpib_of_match);
@@ -319,7 +303,7 @@ static struct platform_driver fluke_keypad_driver = {
         .owner = THIS_MODULE,
         .of_match_table = fluke_keypad_of_match,
     },
-	.probe = fluke_keypad_probe,
+	.probe = &fluke_keypad_probe,
 };
 
 static int fluke_keypad_init(void)
@@ -330,7 +314,7 @@ static int fluke_keypad_init(void)
         printk("fluke_gpib: platform_driver_register failed!\n");
         return result;
     }
-    printK("fluke_keypad\n");
+    printk("fluke_keypad\n");
     return 0;
 }
 
-- 
1.8.4.5

