From 8d559108c43b12db7ed2e60ad92d8d74f6f8d656 Mon Sep 17 00:00:00 2001
From: Alex Dorchak <halfdome@fluke.com>
Date: Thu, 22 Sep 2016 10:14:38 -0700
Subject: [PATCH 32/32] keypad debug continued

---
 drivers/input/keyboard/fluke_keypad.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/input/keyboard/fluke_keypad.c b/drivers/input/keyboard/fluke_keypad.c
index 5dad391..7bdc444 100644
--- a/drivers/input/keyboard/fluke_keypad.c
+++ b/drivers/input/keyboard/fluke_keypad.c
@@ -202,8 +202,6 @@ static int fluke_keypad_probe(struct device *dev)
 	/* Get platform device structure */
 	struct platform_device *pdev = container_of(dev, struct platform_device, dev);
 
-	printk("fluke_keypad%d: base: 0x%08ux irq: %d\n",
-
 	/* Double check resources for sanity */
 	if (pdev->num_resources != 2) {
 		printk(KERN_CRIT "fluke_keypad%d: wrong number of resources for device, got %d.\n",
-- 
1.8.4.5

