reverted:
--- b/src/src.pro
+++ a/src/src.pro
@@ -4,6 +4,9 @@
 include($$OUT_PWD/corelib/qtcore-config.pri)
 include($$OUT_PWD/gui/qtgui-config.pri)
 
+force_bootstrap|!qtConfig(commandlineparser): \
+    CONFIG += force_dbus_bootstrap
+
 src_qtzlib.file = $$PWD/corelib/qtzlib.pro
 src_qtzlib.target = sub-zlib
 
@@ -34,11 +37,13 @@
 
 src_tools_qdbusxml2cpp.subdir = tools/qdbusxml2cpp
 src_tools_qdbusxml2cpp.target = sub-qdbusxml2cpp
+force_dbus_bootstrap: src_tools_qdbusxml2cpp.depends = src_tools_bootstrap_dbus
+else: src_tools_qdbusxml2cpp.depends = src_dbus
-src_tools_qdbusxml2cpp.depends = src_dbus
 
 src_tools_qdbuscpp2xml.subdir = tools/qdbuscpp2xml
 src_tools_qdbuscpp2xml.target = sub-qdbuscpp2xml
+force_bootstrap: src_tools_qdbuscpp2xml.depends = src_tools_bootstrap_dbus
+else: src_tools_qdbuscpp2xml.depends = src_dbus
-src_tools_qdbuscpp2xml.depends = src_dbus
 
 src_winmain.subdir = $$PWD/winmain
 src_winmain.target = sub-winmain
reverted:
--- b/src/tools/qdbuscpp2xml/qdbuscpp2xml.pro
+++ a/src/tools/qdbuscpp2xml/qdbuscpp2xml.pro
@@ -1,6 +1,7 @@
 option(host_build)
 QT = core-private
+force_bootstrap: QT += bootstrap_dbus-private
+else: QT += dbus-private
-QT += dbus-private
 DEFINES += QT_NO_CAST_FROM_ASCII QT_NO_FOREACH
 QMAKE_CXXFLAGS += $$QT_HOST_CFLAGS_DBUS
 
reverted:
--- b/src/tools/qdbusxml2cpp/qdbusxml2cpp.pro
+++ a/src/tools/qdbusxml2cpp/qdbusxml2cpp.pro
@@ -1,6 +1,9 @@
 option(host_build)
+!force_bootstrap:!qtConfig(commandlineparser): \
+    CONFIG += force_bootstrap
 QT = core-private
+force_bootstrap: QT += bootstrap_dbus-private
+else: QT += dbus-private
-QT += dbus-private
 DEFINES += QT_NO_CAST_FROM_ASCII QT_NO_FOREACH
 QMAKE_CXXFLAGS += $$QT_HOST_CFLAGS_DBUS
 
