--- a/configure.ac	2018-11-18 20:02:44.335813588 -0800
+++ b/configure.ac	2018-11-18 20:03:41.315598047 -0800
@@ -768,11 +768,11 @@
   fi
 
   SEARCH_FOR="/include/valgrind/valgrind.h"
-  for i in $SEARCH_PATH ; do
-    if test -r $i/$SEARCH_FOR; then
-      VALGRIND_DIR=$i
-    fi
-  done
+#  for i in $SEARCH_PATH ; do
+#    if test -r $i/$SEARCH_FOR; then
+#      VALGRIND_DIR=$i
+#    fi
+#  done
 
   if test -z "$VALGRIND_DIR"; then
     AC_MSG_RESULT([not found])
